js-packages/meep-sandbox-ctrl-client/src/model/ApplicationInfo.js

```diff
@@ -78,14 +78,12 @@
         obj.id = ApiClient.convertToType(data['id'], 'String');
       if (data.hasOwnProperty('name'))
         obj.name = ApiClient.convertToType(data['name'], 'String');
-      if (data.hasOwnProperty('type'))
-        obj.type = ApplicationType.constructFromObject(data['type']);
-      if (data.hasOwnProperty('state'))
-        obj.state = ApplicationState.constructFromObject(data['state']);
       if (data.hasOwnProperty('mepName'))
```

Comment:
refer to `node-name` instead of `mep-name`

Reply:
Fixed in PR #327